lstchain/calib/camera/flatfield.py

```diff
@@ -240,17 +255,30 @@ def setup_sample_buffers(self, waveform, sample_size):
     def collect_sample(self, charge, pixel_mask, arrival_time):
         """Collect the sample data"""
 
-        # extract the charge of the event and
-        # the peak position (assumed as time for the moment)
+        # verify the time std to reject False FF events
+        # time std of FF events is ~ 1.5 ns for the correct gain channel
+
+        good_time = np.ma.array(arrival_time, mask=pixel_mask)
+        std_arrival_time = np.ma.std(good_time, axis=1)
+        good_FF = (std_arrival_time[HIGH_GAIN] and std_arrival_time[LOW_GAIN]) < self.time_std_cut_outliers
```

Comment:
At the moment, this will basically only check the low gain, since:

```
In [1]: a = 5

In [2]: b = 3

In [3]: a and b
Out[3]: 3

In [4]: (a and b) < 4
Out[4]: True
```

Reply:
but let me verify the logic (it is indeed not what I meant probably)

Reply:
>  at least one of the channel should be < cut (

Then you want `or`, not `and`. But connecting the two stds with and is certainly not what you want

Reply:
```
good_FF = (
    (std_arrival_time[HIGH_GAIN] < self.time_std_cut_outliers)
     or (std_arrival_time[LOW_GAIN] < self.time_std_cut_outliers)
)
```

Reply:
yes, it was wrong (actually I changed the logic at certain point and somehow remained an hybrid version :-) ), is it ok now?

Reply:
yes, it looks ok now.